packages/ui/components/form/inputs/Input.tsx

```diff
@@ -187,6 +190,40 @@ export const PasswordField = forwardRef<HTMLInputElement, InputFieldProps>(funct
   );
 });
 
+export const KeyField = forwardRef<HTMLInputElement, InputFieldProps>(function KeyField(props, ref) {
```

Comment:
Create a key field so we can set the type as password and read-only until the user wants to edit the key.